lib/extended-css-selector.js

```diff
@@ -227,6 +237,8 @@ var ExtendedSelector = (function () { // jshint ignore:line
         }
     };
 
+    var addQuotes = function(_, c1, c2) { return ':' + c1 + '("' + c2.replace(/["\\]/g, '\\$&') + '")'; };
```

Comment:
Every function must have a comment.

This one for instance:
`Used for pre-processing pseud-classes values (see prepareSelector)`